stock_picking_report_valued/readme/MAINTAINER.rst

```diff
@@ -0,0 +1,12 @@
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```

Comment:
I think this file is not necessary, see https://github.com/OCA/maintainer-tools/tree/master/template/module